share/revng/test/configuration/revng/runtime.yml

```diff
@@ -12,6 +12,21 @@ commands:
       ( grep RUN $SOURCE || true ) | sed 's|/\* RUN-\(.*\): \(.*\) \*/|\1,\2|' | while IFS=',' read -r NAME ARGUMENTS; do
           qemu-$QEMU_NAME $INPUT $$ARGUMENTS > $OUTPUT/$$NAME.stdout || true;
       done
+  - type: revng.runtime-abi-test
+    from:
+      - type: revng-qa.runtime-abi-tests
+        filter: "!x86-64 and !i386"
+    suffix: .stdout.yml
+    command: qemu-$QEMU_NAME ${INPUT}/runner-binary ${INPUT}/mmapped-binary > $OUTPUT
```

Comment:
@aleclearmind Do names `runner-binary` and `mmapped-binary` sound good to you?
(this is the first time they explicitly appeared outside of docs)

Reply:
I propose:
* `runner-binary -> runner`
* `mmapped-binary -> compiler-artifact` to stress that we do not intend to run `compiler-artifact`.

Reply:
`runner`: 👌 

`compiler-artifact`: I just weeded out the `artifact` naming from the suite - since it seems to be quite an overloaded term (kind of like `data`). But I do agree with your sentiment. Maybe something along that venue of `test-bodies` or `tested-instructions/functions`, would work better?

Reply:
> artifact [...] seems to be quite an overloaded term

Right.

What about `run-natively` and `foreign-executable`?
So you have `./run-natively foreign-executable`, which kinda succinctly conveys what's happening.

Reply:
Sounds perfect!

Reply:
I guess the only complaint I have now is about the `natively` wording. I'd rather it convey the fact that it modifies the control flow by injecting interrupts instead. Maybe something like `run-with-breakpoints` or `run-while-dumping-the-state`? I agree that it's very verbose, but it better conveys what's hapenning.

Reply:
I think `run-natively` is good enough, otherwise we can have `trace-functions` or `probe-functions`.

Reply:
`trace/probe-calls`?

Reply:
I'd go for `probe-functions` or `probe-calls`.